specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,450 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/restorableDatabaseAccounts/{instanceId}/restorableSqlDatabases": {
+      "get": {
```

Comment:
Can we move API(sql/mongo) specific APIs to a new swagger file ? This way when we move from preview to stable the transition would be easier, plus the current file has grown immensely. You can refer the definitions from cosmos-db.json into the new swagger file.

Reply:
moved to a separate restorable.json file